src/sonic-config-engine/tests/sample_output/frr.conf

```diff
@@ -29,93 +31,55 @@ link-detect
 interface PortChannel04
 link-detect
 !
-!
-! Set ip source to loopback for bgp learned routes
-route-map RM_SET_SRC permit 10
-    set src 10.1.0.32
-!
- 
-route-map RM_SET_SRC6 permit 10
-    set src fc00:1::32
-!
-ip protocol bgp route-map RM_SET_SRC
-!
-ipv6 protocol bgp route-map RM_SET_SRC6
-!
 !!
 !
 ! set static default route to mgmt gateway as a backup to learned default
 ip route 0.0.0.0/0 10.0.0.1 200
 !!
 !
+! template: bgpd/bgpd.main.conf.j2
 !
 ! bgp multiple-instance
 !
-route-map FROM_BGP_SPEAKER_V4 permit 10
+! BGP configuration
 !
-route-map TO_BGP_SPEAKER_V4 deny 10
+! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit 10.1.0.32/32
-ipv6 prefix-list PL_LoopbackV6 permit fc00:1::/64
-!
-!
-route-map TO_BGP_PEER_V4 permit 100
-!
-route-map TO_BGP_PEER_V6 permit 100
-!
-route-map FROM_BGPMON deny 10
-!
-route-map TO_BGPMON permit 10
 !
+ipv6 prefix-list PL_LoopbackV6 permit fc00:1::/64
 !
-route-map ISOLATE permit 10
-  set as-path prepend 65100
 !
-route-map set-next-hop-global-v6 permit 10
-  set ipv6 next-hop prefer-global
```

Comment:
why is this removed?

Reply:
Because now it should be a part of routing-policy.conf.j2 files.
We could check that option in route-map to give warning to users.

Reply:
where can I find this file? I cannot find it.

Reply:
Here
https://github.com/Azure/sonic-buildimage/blob/ea88cfd15b5922c68df2554ea358aa2064b9f36c/dockers/docker-fpm-frr/frr/bgpd/templates/general/policies.conf.j2#L9